README.md

```diff
@@ -75,6 +75,17 @@ cd build
 ctest -C Debug
 ```
 
+### Additional tasks for Pull Requests

```

Comment:
>### [](http://example.com/codeflow?start=0&length=3)

Should we move the whole new section to the Contributing.md?

Reply:
Sounds good, done.